poll-billing.php

```diff
@@ -130,4 +139,14 @@ function CollectData($bill_id) {
     CollectData($argv[1]);
 }
 
+$poller_end  = microtime(true);
```

Comment:
Not sure there's enough functionality here to be worth refactoring, it's only time = end - start and formatting!